lib/tests/test_merged_tree.rs

```diff
@@ -1182,3 +1183,153 @@ fn test_merge_simplify_result() {
+  `revsets.immutable-heads`. For example, set it to `"main"` to prevent
+  rewriting commits on the `main` branch.
+
+### Fixed bugs
+"#;
+    let text4 = r#"### New features
+
+* The `ancestors()` revset function now takes an optional `depth` argument
+  to limit the depth of the ancestor set. For example, use `jj log -r
+  'ancestors(@, 5)` to view the last 5 commits.
+
+* Support for the Watchman filesystem monitor is now bundled by default. Set
+  `core.fsmonitor = "watchman"` in your repo to enable.
+
+* You can now configure the set of immutable commits via
+  `revsets.immutable-heads`. For example, set it to `"main"` to prevent
+  rewriting commits on the `main` branch. The new `immutable()` revset resolves
+  to these immutable commits.
+
+### Fixed bugs
+"#;
+    let text5 = r#"### New features
+
+* The `ancestors()` revset function now takes an optional `depth` argument
+  to limit the depth of the ancestor set. For example, use `jj log -r
+  'ancestors(@, 5)` to view the last 5 commits.
+
+* Support for the Watchman filesystem monitor is now bundled by default. Set
+  `core.fsmonitor = "watchman"` in your repo to enable.
+
+* You can now configure the set of immutable commits via
+  `revsets.immutable-heads`. For example, set it to `"main"` to prevent
+  rewriting commits on the `main` branch.
+
+* `jj op log` now supports `--no-graph`.
+
+### Fixed bugs
+"#;
+    let text6 = r#"### New features
+
+* The `ancestors()` revset function now takes an optional `depth` argument
+  to limit the depth of the ancestor set. For example, use `jj log -r
+  'ancestors(@, 5)` to view the last 5 commits.
+
+* Support for the Watchman filesystem monitor is now bundled by default. Set
+  `core.fsmonitor = "watchman"` in your repo to enable.
+
+* You can now configure the set of immutable commits via
+  `revsets.immutable-heads`. For example, set it to `"main"` to prevent
+  rewriting commits on the `main` branch. The new `immutable()` revset resolves
+  to these immutable commits.
+
+* `jj op log` now supports `--no-graph`.
+
+### Fixed bugs
+"#;
+
+    // conflict in parent commit
+    let base_base = create_single_tree(repo, &[(&conflict_path, text1)]);
+    let base_left = create_single_tree(repo, &[(&conflict_path, text2)]);
+    let base_right = create_single_tree(repo, &[(&conflict_path, text3)]);
+    let base_merged = MergedTree::new(Merge::new(vec![base_base], vec![base_left, base_right]));
+
+    // different conflict in child
+    let left_base = create_single_tree(repo, &[(&other_path, "left"), (&conflict_path, text1)]);
+    let left_left = create_single_tree(repo, &[(&other_path, "left"), (&conflict_path, text2)]);
+    let left_right = create_single_tree(repo, &[(&other_path, "left"), (&conflict_path, text4)]);
+    let left_merged = MergedTree::new(Merge::new(vec![left_base], vec![left_left, left_right]));
+
+    // resolved state
+    let right = create_single_tree(repo, &[(&conflict_path, text5)]);
```

Comment:
Could this be "child1" and "child2" instead of "left" and "right"? Then, there wouldn't be a notion fo `left_right`.

Reply:
Done, and also renamed "base" to "parent".